### ISSUE TITLE
[MasterScript.ahk] make right click open context menus with GUIs that duplicate the existing buttons

### ISSUE THREAD

**Issue by Drugoy** [Drugoy/Autohotkey-scripts-.ahk#4]
Rightclicking items in all the listviews/treeviews should invoke opening a context menu with the menuitems that do the same what toolbar buttons do.

**Comment by WizardDaHacker** [comment]
You have built an amazing program that you should be proud of!

Very useful.

**Comment by Drugoy** [comment]
Thanks for your kind words.
I honestly thought no one used it, so I mainly built it for myself but yeah, I'm proud of the result: without MasterScript it's hard to manage ahk scripts once you realize you have >10 running ahk scripts.

**Comment by WizardDaHacker** [comment]
Of course. Your project should be known across GitHub. I hope you keep on updated this amazing program. It’s very helpful to me and I believe it will be to any other coder.

**Comment by WizardDaHacker** [comment]
Ok. Thanks!
